Monika After Story/game/script-topics.rpy

```diff
@@ -12686,6 +12686,36 @@ label monika_enjoyingspring:
         m 2rkc "...but I guess there's no real way to avoid it, is there?"
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_japanesefood",category=['misc'],prompt="Japanese food",random=True))
+
+label monika_japanesefood:
+    m 2eua "Hey [player], have you ever eaten some kind of japanese food?"
```

Comment:
Capital letter is always used when referring to nationality.

Reply:
Fixed! Thanks for the help.